kalite/distributed/templates/distributed/base.html

```diff
@@ -224,6 +224,9 @@
         </div> {# End wrapper #}
         {# Footer #}
         {% include "distributed/partials/_footer.html" %}
+
+        <script type="text/javascript" src="{% static 'js/tota11y.min.js' %}"></script>
```

Comment:
@radinamatic See here for an example of how to only load this script when DEBUG is true: https://github.com/radinamatic/ka-lite/blob/tota11y/kalite/distributed/templates/distributed/base.html#L69

@aronasorman As I start to play around with the browserify refactor, I will look at how to exclude stuff like this from the built Javascript.